### ISSUE TITLE
{Linked:Bug10209448} [Adaptive Cards > Designer]: Screen reader is announcing incorrect role for the list items present in the select a building dialog.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#8302]
### Target Platforms

Other

### SDK Version

1.5

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
URL: https://adaptivecards.io/designer/
OS Version: 22H2(OS Build 25290.1010)
Browser Version: Version 111.0.1660.7 (Official build) dev (64-bit)
Screen Reader: NVDA

**Repro Steps:**
1. Open the above URL in edge dev browser.
2. Navigate to the 'Card Payload Editor' and press enter key to activate it.
3. Paste the attached JSON file below.
4. Navigate to Preview Mode and press enter key to activate it.
5. Navigate to the List item present under 'Select a Building' list item and observe the issue.

**Actual Result:**
Screen reader is announcing incorrect role for the list items present in the select a building dialog.
Screen reader is announcing as **“Item 1 of 443, 5 button”**.

**Expected Results:**
Screen reader should announce the correct role for the list items present in the select a building dialog.
Screen reader should announce as **“Select a building list 5 selected/not selected 1 of 443"**.

**User Imapct:**
Visually challenged users who rely on Screen readers are getting impacted as Screen reader is announcing incorrect role for the list items present in the select a building dialog.

**MAS Reference:**
https://microsoft.sharepoint.com/sites/accessibility/_layouts/15/Doc.aspx?sourcedoc=%7B507B5D90-F1B9-4AE1-91D1-2F1F2D6C7C1D%7D&file=MAS%204.1.2%20%E2%80%93%20Name%2C%20Role%2C%20Value.docx&action=default&mobileredirect=true&CT=1676004037436&OR=ItemsView

Have feedback to share on Bugs? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-chisha%40microsoft.com%7c1334b110f98744a7f5df08d968eb842b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656183104980596%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=r4ndSHk/LrBDwzi7FO%2BjKN0DCnqUSUcELBvGRkgc5CU%3D&reserved=0)"

### Screenshots

https://user-images.githubusercontent.com/95913748/218003732-0e6b00e1-1dc9-4249-bec8-78b0fa21178d.mp4

[PickerView.txt](https://github.com/microsoft/AdaptiveCards/files/10703785/PickerView.txt)

### Card JSON

```JSON
Attached
```

**Comment by vagpt** [comment]
This issue can be originally tracked using below URL.

https://microsoftit.visualstudio.com/OneITVSO/_workitems/edit/10209448

**Comment by vagpt** [comment]
Closing this issue as per the comment mentioned in the Dependent bug.

I  disagree on the expected result mentioned above announcing “Select a building list 109 selected/not selected 1 of 443" on every item.
As the user laned on this view, it already announced select a building. The user is expected to tap thru the list items and select only one, so there is no selected/not selected state
